es-ds-components/.eslintrc.json

```diff
@@ -0,0 +1,133 @@
+{
+    "parser": "vue-eslint-parser",
+    "parserOptions": {
+        "parser": "@typescript-eslint/parser"
+    },
+    "settings": {
+        "import/resolver": {
+            "typescript": {
+                "alwaysTryTypes": true
+            },
+            "alias": {
+                "map": [
+                    ["@", "./lib"],
```

Comment:
This look like it was copied from https://github.com/EnergySage/es-storyblok-shared/blob/main/.eslintrc.json, I think the paths need to be tweaked for this repo

Reply:
@tomleo 
alias mapping causes this error in es docs, even after tweaked
`"alias": {
                "map": [
                    ["@", "../"]
                ]
            }`
removing it seems to get rid of that erorr.

also this error is the last remaining item
` error  Unable to resolve path to module '@energysage/es-ds-components/components/es-form-msg.vue?raw'  import/no-unresolved` not sure what this is.